recipe/meta.yaml

```diff
@@ -41,9 +44,9 @@ requirements:
   run:
     - python
     - {{ pin_compatible("numpy") }}
-    - numpy <2
+    - numpy >1.23
```

Comment:
I think this line is redundant now and `pin_compatible("numpy")` should be sufficient (even if it wasn't redundant, the constraint is incorrect: it should be `>=1.23`)

xref https://conda-forge.org/docs/maintainer/knowledge_base/#building-against-numpy